pkg/route/engine.go

```diff
@@ -558,13 +558,21 @@ func (engine *Engine) ServeStream(ctx context.Context, conn network.StreamConn)
 
 func (engine *Engine) initBinderAndValidator(opt *config.Options) {
 	// init validator
-	engine.validator = binding.DefaultValidator()
 	if opt.CustomValidator != nil {
 		customValidator, ok := opt.CustomValidator.(binding.StructValidator)
 		if !ok {
 			panic("customized validator can not implement binding.StructValidator")
```

Comment:
"customized validator can not implement binding.StructValidator" -> "customized validator does not implement binding.StructValidator"